src/pages/lobby/[id]/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { useRouter } from "next/router";
+import useStore from "../../../ions/useStore";
+
+const Page = () => {
+	const name = useStore(state => state.name);
```

Comment:
I don't know Zustand, but this variable is lost, when you reload the page. what you think about storing it in local storage?

Reply:
I want to store it in localStorage in the near future. For now I just wanted to see if the name is shown on the next page.